pallets/block-rewards/src/migrations.rs

```diff
@@ -10,144 +10,287 @@
 // MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 // GNU General Public License for more details.
 
-use cfg_traits::rewards::CurrencyGroupChange;
+use cfg_traits::TimeAsSecs;
 use frame_support::{
 	dispatch::GetStorageVersion,
 	inherent::Vec,
 	pallet_prelude::{StorageVersion, Weight},
 	traits::{Get, OnRuntimeUpgrade},
 };
 #[cfg(feature = "try-runtime")]
-use sp_runtime::DispatchError;
-use sp_runtime::{BoundedVec, SaturatedConversion};
-use sp_std::marker::PhantomData;
+use num_traits::Zero;
+use parity_scale_codec::{Decode, Encode};
+use sp_runtime::FixedPointNumber;
 #[cfg(feature = "try-runtime")]
-use {
-	cfg_traits::rewards::AccountRewards,
-	num_traits::Zero,
-	parity_scale_codec::{Decode, Encode},
-};
+use sp_runtime::TryRuntimeError;
+use sp_std::marker::PhantomData;
 
-use crate::{ActiveSessionData, Config, Pallet, SessionData};
-
-pub struct InitBlockRewards<T, CollatorReward, TotalReward>(
-	PhantomData<(T, CollatorReward, TotalReward)>,
-);
-
-fn get_collators<T: pallet_collator_selection::Config>() -> Vec<T::AccountId> {
-	let candidates = BoundedVec::<
-		T::AccountId,
-		<T as pallet_collator_selection::Config>::MaxCandidates,
-	>::truncate_from(
-		pallet_collator_selection::Pallet::<T>::candidates()
-			.into_iter()
-			.map(|c| c.who)
-			.collect(),
-	);
-	pallet_collator_selection::Pallet::<T>::assemble_collators(candidates)
+use crate::{pallet, Config, Pallet, SessionData};
+
+fn inflation_rate<T: Config>(percent: u32) -> T::Rate {
+	T::Rate::saturating_from_rational(percent, 100)
 }
 
-impl<T, CollatorReward, TotalReward> OnRuntimeUpgrade
-	for InitBlockRewards<T, CollatorReward, TotalReward>
-where
-	T: frame_system::Config + Config + pallet_collator_selection::Config,
-	<T as Config>::Balance: From<u128>,
-	CollatorReward: Get<u128>,
-	TotalReward: Get<u128>,
-{
+pub mod init {
```

Comment:
I do not see this migration used. Is it maintained in case we want to initialize another block reward pallet for a runtime?

Reply:
Yes, exactly!